### ISSUE TITLE
`apollo-runtime-kotlin` conflicts with `apollo-runtime`

### ISSUE THREAD

**Issue by rasmusandersson5** [apollographql/apollo-android#2988]
Hi!

I'm trying to add a KMM module with Apollo to a project that contains other (android) modules that is also using Apollo

The KMM module define
`implementation("com.apollographql.apollo:apollo-runtime-kotlin:2.5.2")`

And some of the other android modules define
```
implementation ("com.apollographql.apollo:apollo-android-support:2.5.2")
implementation ("com.apollographql.apollo:apollo-runtime:2.5.2")
implementation("com.apollographql.apollo:apollo-rx2-support:2.5.2")
implementation ("com.apollographql.apollo:apollo-coroutines-support:2.5.2")
```

Each module compiles fine but when compiling the app it fails due to duplicated classes.
Any guideline on how to fix this?

```
Execution failed for task ':App:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class com.apollographql.apollo.ApolloCall found in modules jetified-apollo-runtime-2.5.2 (com.apollographql.apollo:apollo-runtime:2.5.2) and jetified-apollo-runtime-kotlin-jvm-2.5.2 (com.apollographql.apollo:apollo-runtime-kotlin-jvm:2.5.2)
   ...
   ...
```

**Comment by martinbonnin** [comment]
Aww yea, that situation isn't great. Until we can unify both runtimes, your best bet here is to relocate `apollo-runtime-kotlin` using a Gradle plugin such as the shadow plugin: https://imperceptiblethoughts.com/shadow/configuration/relocation/

**Comment by jarmstrong** [comment]
Hi, also struggling with this issue. Could you show how to use ShadowJar for this purpose?

**Comment by martinbonnin** [comment]
Hi @jarmstrong @rasmusandersson5. Apollo Android `3.0.0-alpha01` is out and has [a ton of multiplatform additions](https://github.com/apollographql/apollo-android/releases/tag/v3.0.0-alpha01). 

It also has a new `com.apollographql.apollo3` package so you can use it together with Apollo Android 2. The API should be close enough to Apollo Android 2 but in case you need it, there's a migration guide there: https://www.apollographql.com/docs/android/v3/migration/3.0/.

Let us know how that works.

**Comment by martinbonnin** [comment]
Closing as fixes in `3.0.0-alpha01`
